src/html/login.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" href="css/login.css">
+
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
+	<title>Login</title>
+</head>
+<body>
+    
+<section id="form">
+    <header>
+        <h1>Login</h1>
+    </header>
+    <main>
+        <div id="form-tabs">
+            <div id="form-tab-signin" class="form-tab" name="chosen">Sign in</div>
+            <div id="form-tab-signup" class="form-tab">Sign up</div>
+            <div id="form-tab-line"></div>
+        </div>
+        <div id="form-sections">
+            <section id="form-signin">
+                <div id="signinForm">
+                    <div class="input">
+                        <h3>E-mail</h3>
+                        <input type="text" placeholder="E-mail" id="signin-email"/>
+                    </div>
+                    <div class="input">
+                        <h3>Password</h3>
+                        <input type="password" placeholder="Password" id="signin-password"/>
+                    </div>
+
+                    <div class="additions">
+                        <button id="button-signin">Sign in</button>
+                    </div>
+                </div>
+            </section>
+            <section id="form-signup" style="left: 100%;">
+                <div id="form-signup-inputs">
+                    <div class="input">
+                        <h3>Email</h3>
+                        <input type="email" id="signup-email" placeholder="E-mail">
+                    </div>
+
+                    <div class="input">
+                        <h3>Password</h3>
+                        <input type="password" id="signup-password" placeholder="Password">
+                    </div>
+
+                    <div class="input">
+                        <h3>Confirm Password</h3>
+                        <input type="password" id="signup-password2" placeholder="Confirm Password">
+                    </div>
+
+                    <div class="additions">
+                        <button id="button-signup">Sign Up</button>
+                        <p id="button-signup-verify">Already signed? Validate!</p>
+                    </div>
+                </div>
+
+                <div id="form-signup-verify" style="display: none;">
+                    <div>
+                        <div class="input">
+                        <h3>Email</h3>
+                        <input type="email" id="verify-email" placeholder="E-mail">
+                    </div>
+
+                    <div class="input">
+                        <h3>Verification Code</h3>

```

Comment:
Ponawiam, bo chyba strona główna jest po polsku

Reply:
To chyba nie ten pr...

Reply:
Generalnie, aplikacja jest tworzona po polsku czy angielsku? Bo języki na widokach trochę się różnią